app/src/main/java/ar/com/wolox/android/cookbook/notifications/model/NotificationModels.kt

```diff
@@ -1,38 +1,50 @@
 package ar.com.wolox.android.cookbook.notifications.model
 
 import android.app.PendingIntent
-import android.graphics.Bitmap
+import android.support.annotation.DrawableRes
 
-sealed class NotificationModel(
-    val channelId: String?,
+sealed class SkeletalNotification(
+    val channelId: String,
     val title: String,
     val content: String,
     val priority: Int,
-    val actions: MutableList<NotificationAction> = mutableListOf()
+    var actions: List<NotificationAction> = listOf()
```

Comment:
Why was this made a `var`? Can't you just initialize it correctly and use a `val`?

Reply:
This should be a `val` `List`.